SecurityService/Startup.cs

```diff
@@ -212,6 +224,28 @@ public void ConfigureContainer(ServiceRegistry services)
 
         private void ConfigureMiddlewareServices(IServiceCollection services)
         {
+            services.AddHealthChecks()
+                    .AddSqlServer(connectionString:ConfigurationReader.GetConnectionString("PersistedGrantDbContext"),
+                                  healthQuery:"SELECT 1;",
+                                  name:"Persisted Grant DB",
+                                  failureStatus:HealthStatus.Unhealthy,
+                                  tags:new string[] {"db", "sql", "sqlserver", "persistedgrant"})
+                    .AddSqlServer(connectionString:ConfigurationReader.GetConnectionString("ConfigurationDbContext"),
+                                  healthQuery:"SELECT 1;",
+                                  name:"Configuration DB",
+                                  failureStatus:HealthStatus.Unhealthy,
+                                  tags:new string[] {"db", "sql", "sqlserver", "configuration"})
+                    .AddSqlServer(connectionString:ConfigurationReader.GetConnectionString("AuthenticationDbContext"),
+                                  healthQuery:"SELECT 1;",
+                                  name:"Authentication DB",
+                                  failureStatus:HealthStatus.Unhealthy,
+                                  tags:new string[] {"db", "sql", "sqlserver", "authentication"})
+                    .AddUrlGroup(new Uri($"{ConfigurationReader.GetValue("ServiceAddresses", "MessagingService")}/health"),
+                                 name: "Messaging Service",
+                                 httpMethod: HttpMethod.Get,
+                                 failureStatus: HealthStatus.Unhealthy,
+                                 tags: new string[] { "apllication", "messaging" });
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6239480)